docs/_includes/notify-repo-list.html

```diff
@@ -0,0 +1,11 @@
+<h2>Repository List</h2>
+
+<ul>
+{% for repo in site.notify-repos %}
+  <li>
+    <a href="https://www.github.com/NHSDigital/{{repo.repo-name}}">
```

Comment:
The generated repo page is never linked to

Reply:
Updated to link to the repo pages